src/dune/dune_project.ml

```diff
@@ -348,6 +348,8 @@ module Extension = struct
         -> Univ_map.t * Stanza.Parser.t list
     }
 
+  (* CR-soon amokhov: convert this mutable table to a memoized function, which
+     depends on the contents of dune files that declare extensions. *)
```

Comment:
This is more like the `standard` table for aliases. It's only modified at toplevel.